src/blocks/separator/Separator.types.ts

```diff
@@ -0,0 +1,61 @@
+import { CSSProperties } from 'react';
+
+import { BlocksColors, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import { FlattenSimpleInterpolation } from 'styled-components';
+
+export type SeparatorResponsiveProps = {
+  height?: ResponsiveProp<string>;
+  /* Sets margin css property */
+  margin?: ResponsiveProp<SpaceType>;
+  /* Sets max-height css property */
+  maxHeight?: ResponsiveProp<string>;
+  /* Sets min-height css property */
+  minHeight?: ResponsiveProp<string>;
+  /* Sets max-width css property */
+  maxWidth?: ResponsiveProp<string>;
+  /* Sets min-width css property */
+  minWidth?: ResponsiveProp<string>;
+  /* Sets padding css property */
+  padding?: ResponsiveProp<SpaceType>;
+  /* Sets width css property */
+  width?: ResponsiveProp<string>;
```

Comment:
I don't think we 

- Don't need maxHeight, minHeight css props.  Separator component should be automatically take full available height of the parent component unless given a specific height.
- Don't need maxWidth, minWidth css props.  Separator component should be automatically take full available width of the parent component unless given a specific width.